core/arch/arm/plat-mediatek/main.c

```diff
@@ -16,7 +16,7 @@
 
 static void main_fiq(void);
 
-register_phys_mem(MEM_AREA_IO_NSEC,
+register_phys_mem_pgdir(MEM_AREA_IO_NSEC,
```

Comment:
minor: maybe update indentation at line below.

Reply:
Yep missed that one and same in sunxi. Will fix.